third_party/versions.bzl

```diff
@@ -110,8 +110,8 @@ HTTP_DEPENDENCY_VERSIONS = {
         "version": "0.1.1",
     },
     "rules_jvm": {
-        "sha": "e5b97a31a3e8feed91636f42e19b11c49487b85e5de2f387c999ea14d77c7f45",
-        "version": "2.9",
+        "sha": "f36441aa876c4f6427bfb2d1f2d723b48e9d930b62662bf723ddfb8fc80f0140",
```

Comment:
Can you provide some context on why this upgrade is needed?

Reply:
The older version didn't support the `fail_if_repin_required` attribute. It has been introduced lately, that's why we need this upgrade.

Reply:
Got it--thanks for the clarification!